NEXT_CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## Release v0.247.0
 
 ### Notable Changes
+* Breaking change: Disallow `--` separator in `bundle run` for resource keys ([#2684](https://github.com/databricks/cli/pull/2684))
```

Comment:
What exactly is breaking?

Reply:
I don't think this is the right phrasing for a minor change to behavior that was not previously documented and is likely not in active use by customers.

How about a matter-of-factly entry under CLI changes instead that says something like
```
* Passing a job or pipeline name after the `--` separator in the `run` command now results in an error
```